jsx/brisk_jsx.ml

```diff
@@ -1,6 +1,19 @@
 module P = Ppxlib.Parsetree
 module AT = Ppxlib.Asttypes
 module ATH = Ppxlib.Ast_helper
+module Ast_builder = Ppxlib.Ast_builder.Default
+
+let component_ident ~loc =
+  Ast_builder.(pexp_ident ~loc (Located.lident ~loc "brisk-component"))
+
+let component_ident_pattern ~loc =
+  Ast_builder.(ppat_var ~loc AT.{ txt = "brisk-component"; loc })
```

Comment:
Can we avoid `AT` and use `(Located.lident ~loc "brisk-component")` here? I know I've been using it in a few places but I'd like to remove occurrences rather than multiply.

Reply:
They're not equivalent. `Located.lident` returns a `longident Located.t` while `ppat_var` expects a `string Located.t`...

I'm sure that makes sense somehow, but I'm struggling to find it...

Reply:
Oh, but I can use `Located.mk`!

Reply:
Fixed in https://github.com/briskml/brisk-reconciler/pull/50/commits/d03fea9cffc15625a4374385818b9adb733689b7